packages/x-components/src/x-modules/tagging/service/__tests__/session.service.spec.ts

```diff
@@ -1,46 +1,38 @@
+import { InMemoryStorageService } from '@empathyco/x-storage-service';
 import { DefaultSessionService } from '../session.service';
-import { MockedStorageService } from './utils/mock-storage.service';
 
 describe('testing session id service', () => {
-  const prefix = 'test';
-  const mockedStorageService = new MockedStorageService(prefix);
+  const mockedStorageService = new InMemoryStorageService();
   const sessionService = new DefaultSessionService(mockedStorageService, 1);
+  const storageKey = sessionService.SESSION_ID_KEY;
+
+  const getItemSpy = jest.spyOn(mockedStorageService, 'getItem');
+  const setItemSpy = jest.spyOn(mockedStorageService, 'setItem');
+  const removeItemSpy = jest.spyOn(mockedStorageService, 'removeItem');
 
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
   it('creates a new session id when calling getSessionId and a session does not exist', () => {
     const session = sessionService.getSessionId();
-    expect(mockedStorageService.getPrefix()).toBe(prefix);
-    expect(mockedStorageService.getSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledTimes(1);
-    expect(mockedStorageService.setSessionIdSpy).toHaveBeenCalledWith(
-      'session-id',
-      expect.any(String),
-      expect.any(Number)
-    );
+    expect(getItemSpy).toHaveBeenCalledTimes(1);
+    expect(setItemSpy).toHaveBeenCalledTimes(1);
+    expect(setItemSpy).toHaveBeenCalledWith(storageKey, expect.any(String), expect.any(Number));
     expect(session).toHaveLength(21);
     expect(session).toMatch(/[A-Za-z0-9_-]/);
```

Comment:
Why these two assertions? Do we have any restriction with the values of the sessionId? I can only think of using safe URL characters, but why the 21 characters length?

Reply:
good question. I would only leave the last one.